flow/Arena.cpp

```diff
@@ -1027,7 +1027,7 @@ TEST_CASE("/flow/Arena/Secure") {
 				} else {
 					newBuf = new (arena) uint8_t[len];
 				}
-				ASSERT(newBuf == buf);
+				// ASSERT(newBuf == buf);
```

Comment:
Who is the original author of this? We should get them to weight in. Also, if we don't believe this equality holds, I would remove the assert and associated comment.

Reply:
SGTM.

Reply:
Disabling the assertion would make the remaining checks obsolete.
Please comment out the case entirely until I come up with a code change to reliably, temporarily swap `Arena` alloc/free implementation for the duration of a test run.

Reply:
SGTM. Will update.

Reply:
Done